core/src/main/scala/org/apache/spark/util/AkkaUtils.scala

```diff
@@ -106,7 +104,6 @@ private[spark] object AkkaUtils extends Logging {
       |akka.remote.secure-cookie = "$secureCookie"
       |akka.remote.transport-failure-detector.heartbeat-interval = $akkaHeartBeatInterval s
       |akka.remote.transport-failure-detector.acceptable-heartbeat-pause = $akkaHeartBeatPauses s
-      |akka.remote.transport-failure-detector.threshold = $akkaFailureDetector
```

Comment:
Failure detector threshold was set to a high value to disable it. Since there is not property in akka that let's us do that more idiomatically. By removing the above line we are simply asking akka to pick up default which is very low and will cause workers to disassociate during GC pauses.